### PR TITLE
Larger dashboard

### LINKED ISSUE #3579: Increase the number of dashboard mocks
Generate long dashboards with a significative bigger number of indicators inside.

Currently there are 3 dashboards with 2 indicators in.

### PR DESCRIPTION
**Pull request by Crashillo**
Closes #3579 

## :v: What does this PR do?
Add more indicators to a single dashboard

Checking url: https://diba-plan.gobify.net/planes/pla-de-mandat/2019/dashboards/3

### CODE REVIEW

File: app/javascript/gobierto_dashboards/webapp/components/WidgetIndicatorTable.vue [hunk 1]
```diff
@@ -89,7 +90,7 @@ export default {
     const [y1, y2, y3, y4] = this.values.sort(
       ({ date: a }, { date: b }) => new Date(a) < new Date(b)
     );
-    const years = [...[y1, y2, y3, y4]];
+    const years = [...[y1, y2, y3, y4]].filter(Boolean);
```
Comment (ferblape) [inline app/javascript/gobierto_dashboards/webapp/components/WidgetIndicatorTable.vue:94]: What does this filter do? Maybe a comment would be good

File: app/javascript/gobierto_dashboards/webapp/components/WidgetIndicator.vue [hunk 2]
```diff
@@ -39,7 +39,7 @@
               :value="key"
               required
             >
-            <span>{{ name }}</span>
+            <span>{{ name() }}</span>
```
Comment (ferblape) [inline app/javascript/gobierto_dashboards/webapp/components/WidgetIndicator.vue]: Hmmmm I've seen this code in another PR? Or maybe I already reviewed this PR....

Comment (Crashillo) [reply app/javascript/gobierto_dashboards/webapp/components/WidgetIndicator.vue]: Yeah, it sounds odd. I've merged `master` so this change (which I think it belongs to the previous PR https://github.com/PopulateTools/gobierto/pull/3581/files) shouldn't be here.

File: app/javascript/gobierto_dashboards/webapp/components/WidgetIndicatorTable.vue (hunk 1 above)
Comment (Crashillo) [reply app/javascript/gobierto_dashboards/webapp/components/WidgetIndicatorTable.vue:94]: It's a way to remove `null` or `undefined` from an array. I tested some cases where there weren't four years to parse, but two or three. I'll write it down

Comment (ferblape) [reply app/javascript/gobierto_dashboards/webapp/components/WidgetIndicatorTable.vue:94]: good to know

**Comment by Crashillo** [comment]
Once the tests pass, I'm gonna merge it

**Comment by ferblape** [comment]
Ok 



Enviado desde mi iPhone

File: app/javascript/gobierto_dashboards/webapp/components/WidgetIndicator.vue (hunk 2 above)
Comment (Crashillo) [reply app/javascript/gobierto_dashboards/webapp/components/WidgetIndicator.vue]: I merged again master, now there are only two changed files.

**Pull request merged**